src/scry/symbol.cr

```diff
@@ -114,28 +114,44 @@ module Scry
       node.accept(visitor)
 
       ResponseMessage.new(@text_document.id, visitor.symbols)
+    rescue
+      ResponseMessage.new(@text_document.id, [] of SymbolInformation)
     end
   end
 
   class WorkspaceSymbolProcessor
+    @@crystal_path_files = [] of SymbolInformation
+
     def initialize(@msg_id : Int32 | String, @root_path : String, @query : String)
-      @all_files = Dir.glob("#{root_path}/**/*.cr")
+      @workspace_files = Dir.glob(File.join(root_path, "**", "*.cr"))
+      if @@crystal_path_files.empty?
+        # Memoize crystal stdlib
+        @@crystal_path_files = search_symbols(Dir.glob(File.join(Scry.default_crystal_path, "**", "*.cr")), ".*")
```

Comment:
rename to `crystal_path_symbols` as it stores symbols

Reply:
Done :+1: